library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -41,6 +44,7 @@ sealed class Conversation {
             return when (this) {
                 is V1 -> conversationV1.sentAt
                 is V2 -> conversationV2.createdAt
+                is Group -> TODO()
```

Comment:
I couldn't find a date on the group is that just not exposed yet or am I missing it?

Reply:
I'll fix that